### PR TITLE
Update Techniques and Tactics

### PR DESCRIPTION
**Pull request by 0xzeta**
Summary of the Pull Request
Update Techniques and Tactics

Detailed Description of the Pull Request / Additional comments
Update Techniques and Tactics
- registry_set_persistence_event_viewer_events_asp.yml
- registry_set_winlogon_allow_multiple_tssessions.yml

Example Log Event (In Case of FP Fixes)
NA

Relevant Issues (In Case of Issue Fixes)
NA

### CODE REVIEW

**Comment by nasbench** [comment]
@0xzeta before creating this PR there was a template. Please don't delete it and fill out the necessary information only.

**Comment by 0xzeta** [comment]
Well noted about it

File: rules/windows/registry/registry_set/registry_set_persistence_event_viewer_events_asp.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ date: 2023/02/17
 modified: 2023/03/05
 tags:
     - attack.persistence
+    - attack.defense_evasion
+    - attack.t1547.001
```
Comment (nasbench) [inline rules/windows/registry/registry_set/registry_set_persistence_event_viewer_events_asp.yml:16]: This is `Boot or Logon Autostart Execution: Registry Run Keys / Startup Folder` and these keys are not related to autorun keys.
```suggestion
```

**Review by nasbench — APPROVED** [review]
